bin/porta

```diff
@@ -330,6 +334,32 @@ module Porta
 
     protected
 
+    def docker_internal_host
+      options[:docker_internal_host] ||= containe_local_host_ip
+    end
+
+    def containe_local_host_ip(image: nil)
+      image ||= options[:porxy_image]
+      gw_ip_cmd = %{printf "%d.%d.%d.%d" $(awk '$2 == 00000000 && $7 == 00000000 { for (i = 8; i >= 2; i=i-2) { print "0x" substr($3, i-1, 2) } }' /proc/net/route)}
+      `docker run --rm --entrypoint /bin/bash #{image} -c #{gw_ip_cmd.shellescape}`
+    end
+
+    def container_opts_local_hosts(*hosts)
+      hosts ||= []
```

Comment:
Same as https://github.com/guicassolato/porta-dev-tools/pull/9/files#r633412818

Reply:
I need to have some local array here for the method logic to work. Could you clarify what you don't like? Modifying method parameter? Or the `||=` construct? Or something else? Thanks!